backend/app/service/app.go

```diff
@@ -126,11 +126,8 @@ func (a AppService) PageApp(req request.AppSearch) (interface{}, error) {
 }
 
 func (a AppService) GetAppTags() ([]response.TagDTO, error) {
-	tags, err := tagRepo.All()
-	if err != nil {
-		return nil, err
-	}
-	var res []response.TagDTO
+	tags, _ := tagRepo.All()
+	res := make([]response.TagDTO, 0)
 	for _, tag := range tags {
 		res = append(res, response.TagDTO{
 			Tag: tag,
```

Comment:
The code snippet has several improvements:

1. **Removed Unnecessary Error Handling**: The line `return nil, err` is unnecessary because the `_` variable already captures any error from the call to `tagRepo.All()`.

2. **Optimized Loop Initialization**: Instead of creating a slice with a fixed size and then appending elements, you can directly initialize a new slice and append only needed values during iteration. This avoids the initial allocation size overhead if the loop iterates less than twice.

3. **Use Slice Literal Syntax**: It's cleaner and more idiomatic to use the slice literal syntax `[0]` (`make([]response.TagDTO, 0)`).

4. **Consistent Variable Naming**: While not always necessary for readability, it might be good practice to ensure consistent naming conventions.

Here’s an optimized version of the function:

```go
func (a AppService) GetAppTags() ([]response.TagDTO, error) {
	tags, _ := tagRepo.All()
	res := make([]response.TagDTO, len(tags))
	for i, tag := range tags {
		res[i] = response.TagDTO{
			Tag: tag,
		}
	}
	return res, nil
}
```

These changes improve performance slightly by reducing memory allocations at initialization and making the logic easier to read and maintain.